cardano-node-chairman/src/Testnet/ByronShelley.hs

```diff
@@ -179,7 +179,7 @@ testnet H.Conf {..} = do
     , "maxProposalSize" .= J.toJSON @String "700"
     , "mpcThd" .= J.toJSON @String "20000000000000"
     , "scriptVersion" .= J.toJSON @Int 0
-    , "slotDuration" .= J.toJSON @String "2000"
+    , "slotDuration" .= J.toJSON @String "1000"
```

Comment:
Don't change this one. 2000 milliseconds is 2 seconds, which is what we want for Byron, since there's 1 block per slot for Byron/BFT.

Reply:
Switched this back.